var/spack/repos/builtin/packages/py-numpy/package.py

```diff
@@ -84,7 +84,8 @@ def patch(self):
         def write_library_dirs(f, dirs):
             f.write('library_dirs=%s\n' % dirs)
             if not ((platform.system() == "Darwin") and
-                    (platform.mac_ver()[0] == '10.12')):
+                    (Version(platform.mac_ver()[0]).up_to(2) == Version(
+                        '10.12'))):
```

Comment:
`platform.mac_ver()` usually returns a 3-digit string, which will never be equal to `10.12`

Reply:
`platform.mac_ver` returns `('10.14.5', ('', '', ''), 'x86_64')` for me, so I assume you mean a 3-part version (vs. a 3-digit version). The logic here makes sense in that case.